ilib-webpack-plugin.js

```diff
@@ -448,32 +449,58 @@ function IlibDataPlugin(options) {
 }
 
 IlibDataPlugin.prototype.apply = function(compiler) {
-    compiler.plugin('compilation', function(compilation, callback) {
-        // console.log("@@@@@@@@@@@@@@@@ compilation");
-        compilation.ilibWebpackPlugin = this; // make sure the ilib webpack loaders can find this plugin
-
-        compilation.plugin('finish-modules', function(modules) {
-            // console.log("@@@@@@@@@@@@@@@@ finish-modules");
-            if (localeData.size > 0) {
-                try {
-                    var sources = emitLocaleData(compilation, this.options);
-
-                    // Now update the in-memory modules with these sources because it doesn't
-                    // reread the files on disk that we just wrote out.
-                    modules.forEach(function(module) {
-                        if (sources[module.resource]) {
-                            module._source._value = '"use strict";' + sources[module.resource];
-                        }
-                    });
-                } catch (e) {
-                    console.log("ilib-webpack-plugin: " + e.toString());
-                    throw e;
+    // Webpack 4
+    if (compiler.hooks) {
+        compiler.hooks.compilation.tap(PLUGIN_NAME, (compilation) => {
```

Comment:
Does ES6 code work here?

Reply:
will check! thanks for looking at this.

Reply:
@ehoogerbeets i've updated this. going to try and get a smaller, webpack4 application scaffolded for testing.